plugins/outputs/signalfx/README.md

```diff
@@ -0,0 +1,19 @@
+# SignalFx Output Plugin
+
+```toml
+    ## SignalFx API Token
+    APIToken = "my-secret-key" # required.
+
+    ## Ingest URL
+    DatapointIngestURL = "https://ingest.signalfx.com/v2/datapoint"
+    EventIngestURL = "https://ingest.signalfx.com/v2/event"
+    
+    ## Exclude metrics by metric name
+    Exclude = ["plugin.metric_name", ""]
```

Comment:
What's up with the blank string as the second item?

Reply:
will address